Cargo.toml

```diff
@@ -1,86 +1,34 @@
-[package]
+[workspace]
+resolver = "2"
+
+members = [
+        "examples",
+        "micro-rdk",
+        "micro-rdk-installer",
+        "micro-rdk-macros",
+]
+
+[workspace.package]
 authors = ["Viam, inc. <contact@viam.com>"]
 description = "Viam RDK for microcontroller"
 edition = "2021"
 license = "AGPL-3.0"
-name = "micro-rdk"
 repository = "https://github.com/viamrobotics/micro-rdk"
-resolver = "2"
-version = "0.1.6"
 rust-version = "1.71"
```

Comment:
This will be shared with all crates in the worksapce? Can it be overwritten if we wanted to? (ex may micro-rdk-installer will need a more recent version of rust than the one used by micro-rdk?)